dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -73,41 +73,18 @@
                     = t('teacher.panel.example')
           = link_to build_script_level_path(@script_level, @section? {section_id: @section.id} : {}) do
             %button.btn.btn-info= t('teacher.panel.try_it_yourself')
-        - if @section
-          .section
-            %h4
-              = t('teacher.panel.section')
-              = link_to @section.name, teacher_dashboard_section_progress_url(@section)
-        - if current_user.sections.many?
+        - if current_user.sections.any?
+          #check-mark-container{style: "display: none"}
```

Comment:
This is a little bit gross. We want to use a checkmark in our React component. This check mark is a digested asset in dashboard (that is also use elsewhere). We don't have a fixed path we can use in React, so instead we let dashboard render a hidden div, and the component goes and extracts the src attribute from it. Not sure if there's a cleaner way to accomplish this.